accel-pppd/session.c

```diff
@@ -240,6 +240,11 @@ void __export ap_session_finished(struct ap_session *ses)
 		ses->ifname_rename = NULL;
 	}
 
+	if (ses->vrf_name) {
```

Comment:
alike back-renabe above and netns release below, need to move to default (0) vrf

Reply:
also, ppp unit_cache also need it alike restoring default netns